src/app/docs/inline-delete/inline-delete-docs.component.html

```diff
@@ -0,0 +1,99 @@
+<sky-docs-demo-page
+  moduleName="SkyInlineDeleteModule"
+  moduleSourceCodePath="src/app/public/modules/inline-delete/"
+  pageTitle="Inline delete"
+>
+  <sky-docs-demo-page-summary>
+    Sample description.
+  </sky-docs-demo-page-summary>
+
+  <sky-docs-design-guidelines>
+
+    <sky-docs-demo-page-section
+      heading="Usage"
+    >
+      <sky-docs-design-guideline
+        heading="Use when"
+        headingStyle="success"
+      >
+        <p>
+          Use inline delete when deleting an item from a list.
```

Comment:
Agree about the wording needing to convey confirming delete.

My thought around singular vs plural is that plural implies you'd use this if you used multiselect -> bulk delete, which is not the case.

Reply:
Makes sense. How about:

"Use the inline delete confirmation when users need to confirm that they want to delete an item from a list."

Reply:
Just confirming:
Is the intended usage determination for this _entirely_ about being done from within lists? Is there concern paid to the complexity/significance of the object being deleted? From a design perspective, are we ok with any record, no matter how complex, getting this treatment when being deleted from a list?

Reply:
@Blackbaud-AdamFunderburk If the guidance for complex objects is that they should be deleted from their record page rather than lists referencing them this shouldn't come up, unless you think there are cases where complex objects won't have record pages.

Reply:
I'm think I'm ok with making that our guidance: "Don't delete anything complex enough to have its own record page from a list of those records."

Is it worth calling that our here?

Reply:
I update the Use when item to reflect my suggestion. ... I'm leaving this comment thread unresolved though because we need to figure out where to provide guidance on not deleting items in lists that are complicated enough to have records. I don't think we should include that in a Use when or Don't use when item because this component is just for the delete confirmation, not the delete action. So it seems like that guidance should be highlighted differently.

Reply:
@Blackbaud-AdamFunderburk, I circled back to this, and to address your points here, I added a second paragraph to the "Use when" list:
```Keep in mind that you should not delete items from lists when they are complex enough to have their own record pages. For lists with such items, do not include delete actions in the context menus.```

I also added a link to https://developer.blackbaud.com/skyux/design/guidelines/managing-records in the "Related information" section

Reply:
I also created a PR to add this guidance to the dropdown docs since it seems more pertinent there than here: https://github.com/blackbaud/skyux-popovers/pull/55.

Reply:
Resolving this comment thread after Todd and Trevor signed off on the changes here and in the dropdown docs.